src/codeu/chat/client/commandline/Chat.java

```diff
@@ -49,7 +49,7 @@ private static void help() {
     System.out.println("   sign-out  - sign out current user.");
     System.out.println("   current   - show current user, conversation, message.");
     System.out.println("User commands:");
-    System.out.println("   u-add <name>  - add a new user.");
+    System.out.println("   u-add <name> <password>  - add a new user.");
```

Comment:
In a perfect world with infinite time it would be a good idea to maintain the old command line client, but you don't have to now. You could just get rid of it if you don't want to deal with it anymore.